cpp/include/cugraph/prims/property_op_utils.cuh

```diff
@@ -171,6 +171,30 @@ constexpr auto op_dispatch(raft::comms::op_t op, F&& f)
   };
 }
 
+template <typename T>
+T identity(raft::comms::op_t op)
```

Comment:
I feel like this is a bit of misnomer...

identity sounds like an identity function... I will also think about a better name, but if you have a better one, I think we should rename this.

Reply:
An operator also has the concept of identity.

See :
"Identity element - Wikipedia" https://en.m.wikipedia.org/wiki/Identity_element#Definitions

Basically any element when applied to a binary operator yields the other element, it's called an identity element. If this is confusing then sure we could change the name.

Reply:
https://en.wikipedia.org/wiki/Identity_element

I think we'd better use ```identity_element``` or ```neutral_element```

Reply:
You meant an operand? Yeah.... both ```idenity``` and ```ideneity_element``` can be used, but ```identity_element``` is clearer.

Reply:
How about passing an optional in place of init so that the internal implementation selects the appropriate init when nullopt is passed.

Reply:
Any motivation for that? Neither thrust nor std library does this, so I am negative unless there is a strong motivation for doing that.